examples/ports/main.cc

```diff
@@ -68,9 +68,9 @@ class Adder : public Reactor {
   Reaction r_add{"r_add", 1, this, [this]() { add(); }};
 
 public:
-  Input<int> i1{"i1", this};
-  Input<int> i2{"i1", this};
-  Output<int> sum{"sum", this};
+  Input<int> i1{"i1", this};    // NOLINT
+  Input<int> i2{"i1", this};    // NOLINT
```

Comment:
about what did clang-tidy complain here ?

Reply:
That the fields have public visibility. And I was too lazy to make them private and add getters ;), it's just an example after all